source/kubernetes/security.rst

```diff
@@ -52,7 +53,15 @@ With command line, when creating a new Kubernetes cluster, please use label
 
 .. code-block:: bash
 
-  openstack coe cluster create k8s-1 --merge-labels --labels admission_control_list=PodSecurityPolicy --cluster-template kubernetes-v1.18.2-prod-20200630
+  openstack coe cluster create k8s-1 --merge-labels --labels admission_control_list=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota,PodSecurityPolicy --cluster-template kubernetes-v1.18.2-prod-20200630
+
+.. Note::
+  To override the default admission controller list, you have to provider
+  the full list you want to apply. For instance, in above example, you also
+  need to at least provide admission controller like ServiceAccount, NamespaceLifecycle
+  and etc to make sure it won't break the cluster bootstrap. We recommend to
+  at least include these admission controllers: ``NamespaceLifecycle``, ``LimitRanger``,
+  ``SecurityContextDeny``, ``ServiceAccount``, ``ResourceQuota``.
```

Comment:
Is this the default set?

Reply:
Yep, actually, they are default settings now. See https://review.opendev.org/#/c/748389/